crates/polars-lazy/src/physical_plan/expressions/apply.rs

```diff
@@ -354,10 +354,12 @@ impl PhysicalExpr for ApplyExpr {
                 },
                 ApplyOptions::GroupWise => self.apply_multiple_group_aware(acs, df),
                 ApplyOptions::ElementWise => {
-                    if acs
-                        .iter()
-                        .any(|ac| matches!(ac.agg_state(), AggState::AggregatedList(_)))
-                    {
+                    if acs.iter().any(|ac| {
+                        matches!(
+                            ac.agg_state(),
+                            AggState::AggregatedList(_) | AggState::AggregatedScalar(_)
```

Comment:
I think it can if all `ac`s are `AggregatedScalar`. I think any `NotAggregated` cannot combine with `AggregatedList | AggregatedScalar` (eg `aggregated`).

So if all are `AggregatedScalar` and the groups are equal we could still go int he elementwise, I think